### ISSUE TITLE
[Zigbee2Mqtt] Failed to convert value for device RadiateurSalon: Error: Zigbee2mqqt don't handle value "LOCK" for feature "mode".

### ISSUE THREAD

**Issue by LM1LC3N7** [GladysAssistant/Gladys#1448]
**Describe the bug**
When a device sends a value not compatible with Gladys Zigbee2Mqtt, the zigbee2mqtt service crash
Related to https://github.com/GladysAssistant/Gladys/issues/1426

**logs**

```bash
2022-03-01T12:48:08+0100 <error> handleMqttMessage.js:97 () Failed to convert value for device RadiateurSalon: Error: Zigbee2mqqt don't handle value "LOCK" for feature "mode".
    at convertValue (/src/server/services/zigbee2mqtt/utils/convertValue.js:55:17)
    at /src/server/services/zigbee2mqtt/lib/handleMqttMessage.js:93:26
    at Array.forEach (<anonymous>)
    at Zigbee2mqttManager.handleMqttMessage (/src/server/services/zigbee2mqtt/lib/handleMqttMessage.js:86:41)
    at MqttClient.<anonymous> (/src/server/services/zigbee2mqtt/lib/connect.js:53:12)
    at MqttClient.emit (events.js:400:28)
    at MqttClient._handlePublish (/src/server/services/zigbee2mqtt/node_modules/mqtt/lib/client.js:1277:12)
    at MqttClient._handlePacket (/src/server/services/zigbee2mqtt/node_modules/mqtt/lib/client.js:410:12)
    at work (/src/server/services/zigbee2mqtt/node_modules/mqtt/lib/client.js:321:12)
    at Writable.writable._write (/src/server/services/zigbee2mqtt/node_modules/mqtt/lib/client.js:335:5)
    at doWrite (/src/server/services/zigbee2mqtt/node_modules/readable-stream/lib/_stream_writable.js:409:139)
    at writeOrBuffer (/src/server/services/zigbee2mqtt/node_modules/readable-stream/lib/_stream_writable.js:398:5)
    at Writable.write (/src/server/services/zigbee2mqtt/node_modules/readable-stream/lib/_stream_writable.js:307:11)
    at Socket.ondata (internal/streams/readable.js:731:22)
    at Socket.emit (events.js:400:28)
    at addChunk (internal/streams/readable.js:293:12)
    at readableAddChunk (internal/streams/readable.js:267:9)
    at Socket.Readable.push (internal/streams/readable.js:206:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:188:23)
```

**Comment by Pierre-Gilles** [comment]
I edited the title here, because there is no crash here

**Comment by atrovato** [comment]
This should be fixed with #1383

**Comment by Pierre-Gilles** [comment]
@LM1LC3N7 Can you test the PR of @atrovato and see if it fixes the problem ?

**Comment by LM1LC3N7** [comment]
Sorry I did not see the notification from GitHub :)

After merging all @atrovato zigbee2mqtt edits from "zigbee-set-value" I still have errors.
To merge, here's what I done:

- docker exec -ti gladys sh
- install git
- `git clone https://github.com/atrovato/Gladys -b zigbee-set-value`
- `cp -rf Gladys/server/services/zigbee2mqtt/ services/` (move @atrovato code to current Gladys container code)
- restart container

### error
![image](https://user-images.githubusercontent.com/14235706/157014389-8231836c-c1fd-40dd-b25d-ff89cf430044.png)


### vim /src/server/services/zigbee2mqtt/lib/handleMqttMessage.js
![image](https://user-images.githubusercontent.com/14235706/157014667-dc87ed78-bbc3-4c89-818c-557b868bb44a.png)

**Comment by atrovato** [comment]
Thank you, I had returns from other testers too, kind same error.
I just publish a fix, on the same branch.
Can you try again?

**Comment by LM1LC3N7** [comment]
It appears this time it works well!

![image](https://user-images.githubusercontent.com/14235706/157419308-84cc5ad3-54ef-491e-933b-1575eb8b91dc.png)

**Comment by LM1LC3N7** [comment]
FYI with Gladys 4.8.2 the issue is not solved yet.

![image](https://user-images.githubusercontent.com/14235706/160396073-b5a56ab6-86b1-47f8-ba74-969942bb8f17.png)

**Comment by atrovato** [comment]
No, PR is not merged :/

**Comment by Pierre-Gilles** [comment]
Yes, we had some critical bugs in Gladys recently that needed to be fixed, v4.8.2 was only a small bug fix version, I wanted to keep the version as small as possible so it could go live fast without too much feedbacks needed.

This one is a big one has it impacts all Zigbee2mqtt users and I want to merge this right.

**Comment by Pierre-Gilles** [comment]
FYI, I merged the PR and the fix today, and just started a Docker build on `gladysassistant/gladys:dev` 

https://github.com/GladysAssistant/Gladys/actions/runs/2078334755

I'm looking for testers before I deploy this to prod just to make sure the final version is fine :)
